docs/release-notes/1.8.3.rst

```diff
@@ -5,6 +5,9 @@
 
 .. rubric:: Bug fixes
 
+- Fixed finding variables with ``use_raw=True`` and ``basis=None`` in
+  :func:`scanpy.pl.scatter` :pr:`2027` :small:`I Virshup`
```

Comment:
```suggestion
  :func:`scanpy.pl.scatter` :pr:`2027` :small:`E Rice`
```

Reply:
@esrice, should probably be your name here!

Btw, I can't seem to commit to this branch, I believe because it's been made on your lab repo and not a personal fork. In future, I can fix small stuff like this if I'm able to commit to the branch.